db/migrate/20210617203927_migrate_admin_tax_amounts.rb

```diff
@@ -0,0 +1,52 @@
+class MigrateAdminTaxAmounts < ActiveRecord::Migration[6.0]
+  class Spree::Adjustment < ApplicationRecord
+    belongs_to :originator, polymorphic: true
+    belongs_to :adjustable, polymorphic: true
+    belongs_to :order, class_name: "Spree::Order"
```

Comment:
I don't see this one used either. We don't actually want to load `Spree::Order`. We just use the `order_id` column which doesn't need to know about the Order code.